src/ui/components/ErrorText.test.tsx

```diff
@@ -1,42 +1,49 @@
 import React from 'react';
-import {create} from 'react-test-renderer';
 import {render} from 'src/testUtils';
 import {ErrorText} from './ErrorText';
 
-describe('ErrorText component', () => {
-  it('should render the prop component correctly', async () => {
-    const {getByText} = render(<ErrorText>Mocked Error Text</ErrorText>);
-    expect(getByText('Mocked Error Text')).toBeTruthy();
-  });
+test('should render with children data', async () => {
+  const {findByText} = render(<ErrorText>Mocked Error Text</ErrorText>);
+  expect(await findByText('Mocked Error Text')).toBeTruthy();
```

Comment:
This test was Ok before, why do you need to `await`?

Reply:
Ohh!! Ya, my bad!! will revert it back.